kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-common/src/main/java/org/kie/server/remote/rest/common/util/RestUtils.java

```diff
@@ -118,6 +122,28 @@ public static String getContentType(HttpHeaders headers) {
         return contentType;
     }
 
+    public static MediaType getMediaType(HttpHeaders httpHeaders) {
```

Comment:
Isn't this the exact same method as in the `ControllerUtils` class? If not, ignore me! :)